lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -8438,6 +8461,23 @@ const byte * InterpreterStackFrame::OP_ProfiledLoopBodyStart(const byte * ip)
         return m_inParams + 1;
     }
 
+    ForInObjectEnumerator * InterpreterStackFrame::GetForInEnumerator(uint index)
```

Comment:
>index [](start = 75, length = 5)

This argument really means the for..in loop level, right? If so, we might be better off calling it level instead of index (less confusion to readers). (would also have to change names of params in other methods from forInLoopId to forInLoopLevel)